src/pages/library/DataBrowserPreview.js

```diff
@@ -67,6 +82,21 @@ const DataBrowserPreview = ({ id }) => {
     loadTable()
   }
 
+  const formatTableCell = cellContent => {
+    return Utils.cond(
+      [!Utils.cantBeNumber(cellContent), () => cellContent],
+      [Utils.maybeParseJSON(cellContent), () => {
+        const contentAsJSON = Utils.maybeParseJSON(cellContent)
+        const contentAsPrettyString = JSON.stringify(contentAsJSON, null, 4)
```

Comment:
We have a library - `react-json-view` that does a nice job of prettifying JSON for display. Here is an example with some of our data:
![image](https://user-images.githubusercontent.com/50371603/146190104-afd1fac9-de5f-41a0-9c65-1349208c4ffc.png)

Reply:
wow i love how that looks, oky, let me  update it to this!